src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/FunctionCallUtilities.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
This class shouldn't really be in the "Primitives" assembly, but there's really no other place it could be kept given the current project structure.